init.c

```diff
@@ -100,7 +103,17 @@ utility_hook(Node *parsetree,
 
 	PG_TRY();
 	{
-		standard_ProcessUtility(parsetree, queryString, context, params, dest, completionTag);
+		/* If there's another hook in the chain, pass on the action */
+		/* and hope/pray that it passes it along eventually to the */
```

Comment:
har de har. We are a smidge better about that, after this patch (though we'll still break standard usage of event triggers in favor of our own).